modules/drivers/athena/test/metabase/driver/athena_test.clj

```diff
@@ -81,8 +81,12 @@
             query        (-> (mt/native-query {:query sql, :params args})
                              (assoc-in [:middleware :format-rows?] false))]
         (mt/with-native-query-testing-context query
-          (is (= [#t "2022-11-16T04:21:00.000-08:00[America/Los_Angeles]" #t "05:03"]
-                 (mt/first-row (qp/process-query query)))))))))
+          (let [[ts t] (mt/first-row (qp/process-query query))]
+            (is (#{#t "2022-11-16T04:21:00.000-08:00[America/Los_Angeles]"
+                   #t "2022-11-16T04:21:00.000-08:00[US/Pacific]"}
+                   ts))
+            (is (= #t "05:03"
+                   t))))))))
```

Comment:
belongs elsewhere?

Reply:
Yes, there was a failing Athena test, the fix has been merged in a separate PR.